src/Workflow.py

```
@@ -410,6 +413,243 @@ def execution(self) -> None:
+        params = self.parameter_manager.get_parameters_from_json()
+
+        # Make sure output directory exists
+        base_path = dirname(self.workflow_dir)
+        
+        # Get input files
+        in_mzml = self.file_manager.get_files(self.params["mzML-files"])[0]
+
+        # Generate output folder
+        current_base = splitext(basename(in_mzml))[0]
+        current_time = time.strftime("%Y%m%d-%H%M%S")
+        dataset_id = '%s_%s'%(current_base, current_time)
```

Ruff (0.11.9)

Local variable `dataset_id` is assigned to but never used

Remove assignment to unused variable `dataset_id`

(F841)